x-pack/plugins/security_solution/public/overview/components/overview_risky_host_links/navigate_to_host.tsx

```diff
@@ -34,8 +34,10 @@ export const NavigateToHost: React.FC<{ name: string }> = ({ name }): JSX.Elemen
     [filterManager, name, navigateToApp]
   );
   return (
-    <EuiButtonEmpty color="text" onClick={goToHostPage} size="xs">
-      <EuiText size="s">{name}</EuiText>
-    </EuiButtonEmpty>
+    <EuiToolTip content={name} position="top">
+      <EuiButtonEmpty color="text" onClick={goToHostPage} size="xs">
+        <EuiText size="s">{name}</EuiText>
+      </EuiButtonEmpty>
+    </EuiToolTip>
```

Comment:
Would be nice if we had a component that only showed the tooltip if the content was overflowing (to prevent unneeded tooltips on regular mouse movement), but I don't think there's one floating around.